packages/contextual-menu-react/documentation/ContextualMenu.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Contextual Menu
+react: contextual-menu-react
+scss: contextual-menu
+group: edit me!
+---
+
+import ContextualMenuExample, { contextualMenuExampleCode, contextualMenuExampleKnobs } from "./ContextualMenuExample";
+
+<Ingress>
+    Med kontekstuell meny presenteres en liste med kontekst relatert utvalg (alternativer) på en midlertidig overflate.
+</Ingress>
+
+<ComponentExample
+    component={ContextualMenuExample}
+    codeExample={contextualMenuExampleCode}
+    knobs={contextualMenuExampleKnobs}
+/>
+
+Inneholder et lite sett med relevante handlinger, relatert til en kontroll, et stykk innhold, en visning i en app eller et område av brukergrensesnittet (trigger).
+Menyen vises når brukere samhandler med en trigger (ikon, knapp, tekst, kombo av tekst og ikon f.eks., handling)
+Kontekstuell meny er mindre prominente og tar mindre plass enn Select komponent (digresjon).
+Det er mulig å bygge utvidbar liste men det anbefales å bruke maksimal to nivåer. Bruk gjerne færre enn 10 items i en liste.
+Liste items skal kunne grupperes med skillelinje.
+
+## Mulig innhold i en meny
+
+Tekst, ikon+tekst, knapp > forskjellige interaktive elementer
+Keyboard shortcuts
+
+## Triggers
+
+Desktop
+Klikke på visuell element; Icons: Ellipsis button (h/v), Chevrons, Stykk av tekst, tekst+Chevron
+Ctrl+klikk, høyre mus klikk
+Klikk med to fingre på trackpad
+
+Mobil
+Tapping på visuell element; Icons: Ellipsis button (h/v), Chevrons, Stykk av tekst, tekst+Chevron
+Trigger Dots-icon> Elipsis button  
+Hvis ikonen skal være en del av horisontall serie av elementer, så brukes en horisontal variant.
+Hvis ikonen står alene så er det fritt å velge orjentasjonen h/v
+Plassering av menu liste
+Listen blir plassert under elementet som genererer den (trigger)
+Plasseringen skjer relativt til skjerm slik at listen blir alltid synlig uansett nærheten av skjermkanten.
```

Comment:
Disse er kanskje også lister? Bør sikkert også avsjekkes hvilke av disse som faktisk fungerer som trigger i implementasjonen.

Reply:
Har ryddet opp i dokumentasjon vi ikke har funksjonalitet for per nå.